bpf/bpf_lxc.c

```diff
@@ -1029,27 +1017,17 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx, __u32 *d
 		if (ct_status == CT_REPLY || ct_status == CT_RELATED)
```

Comment:
It seemed like "is this even a request?" is best determined by the caller(s). They have more context, and I expect it wouldn't always need to come from CT. So requiring all callers to pass in a `ct_status` just because this fits for the only user **right now** didn't seem a good fit.

I suppose an alternative would be

```
bool egress_gw_needs_redirect(..., bool is_request, ...)
{
	if (!is_request)
		return false;
	...
}
``` 

And calling that with something like

```
if (egress_gw_needs_redirect(..., ct_is_request(ct_status), ...)) {
	...
}
```

But that doesn't feel much different from just encoding the "needs to be a request" requirement into the function name?